mining/src/manager_tests.rs

```diff
@@ -1116,6 +1117,47 @@ mod tests {
         // TODO: extend the test according to the golang scenario
     }
 
+    #[test]
+    fn test_evict() {
```

Comment:
Please add comments to explain what's happening in this test. Maybe add inline comments.

From reading it, it creates 10 txs and a heavy tx and the expectation is that to accommodate the heavy tx, 6 of the lighter txs are evicted.